index.html

```diff
@@ -1,14 +1,31 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-   <meta charset="UTF-8">
-   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-   <link rel="stylesheet" href="/src/output.css">
-   <title>Baumeister</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/src/output.css">
+    <title>Website project template</title>
```

Comment:
I was wondering why you didn't keep the title as 'Baumeister' since that's the website we are building, plus 'website project template' as a title is too generic.

Reply:
Sorry we forgot to add the tittle. We will be sure to add it